docs/source/advancedcommands.rst

```diff
@@ -233,3 +233,161 @@ Example Usage
 The ``normsub`` argument specifies an expected value for your solution to normalize the different :math:`g_i` (you can also do this by hand). The feasibility of the problem should not depend on the normalization, but the spacing of the sweep will.
 
 The ``sweep`` argument specifies what points between 0 and 1 you wish to sample the weights at. If you want different resolutions or spacings for different weights, the ``sweeps`` argument accepts a list of sweep arrays.
+
+Linking Variables
+=================
+
+When modeling a complex system, it is often desirable to begin by independently modeling each subsystem. This makes debugging and model validation easier. After developing all the subsystem models, they can be linked together to form a comprehensive system model. For example, instead of having one large airplane model, there may be separate wing, fuselage, vertical tail, horizontal tail, engine, and landing gear models which are linked together to form a full airplane model.
```

Comment:
That's true. A lot of things seem to start as monolithic scripts / spreadsheets and get broken up for ease of reuse/maintenance/divided responsibility.